### PR TITLE
1.6

### PR DESCRIPTION
**Pull request by khaledAbbas91**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/writing-running-appium/network_connection.md [hunk 1]
```diff
@@ -17,8 +17,19 @@ connection API for iOS.
...
 ### Android
 
-Choose the setting you would like to use, and then send the correct bitmask from
-the table above.
+There are the following limitations:
+
+#### Real Devices
+
+* Changing Airplane Mode state only works for Android 6 and older
+* Chaning data connection state works for Android 4.4 and older. Newer OS releases (5.0+) must be rooted in order to make the feature working (e. g. _su_ binary should available)
```
Comment (mykola-mokhnach) [inline docs/en/writing-running-appium/network_connection.md]: typo: Changing

File: lib/main.js [hunk 2]
```diff
@@ -48,9 +48,32 @@ function logDeprecationWarning (deprecatedArgs) {
...
 
 function logNonDefaultArgsWarning (args) {
+  // cleanly print out the default arguments, allowing for prefix and timestamp
+  function getValueArray (obj, indent = '  ') {
+    if (!_.isObject(obj)) {
+      return [obj];
+    }
+
+    let strArr = ['{'];
+    for (let [arg, value] of _.toPairs(obj)) {
```
Comment (mykola-mokhnach) [inline lib/main.js]: Shouldn't one use _.isPlainObject verification to make _.toPairs work as expected?

File: lib/main.js [hunk 3]
```diff
@@ -132,6 +162,17 @@ async function main (args = null) {
     await server.close();
     throw err;
   }
+
+  process.once('SIGINT', async function () {
```
Comment (mykola-mokhnach) [inline lib/main.js]: why not to use lambda declaration?

**Comment by imurchie** [comment]
I think this is submitted in error. Closing.

**Pull request closed without merging**